app/app.go

```diff
@@ -137,6 +137,9 @@ import (
 const (
 	AccountAddressPrefix = "elys"
 	Name                 = "elys"
+
+	// Temperal declaration until we have dex module
+	DexRevenuCollectorName = "dev_rev_collector"
```

Comment:
Can you use full name not shortend ? `DexRevenueCollectorName`

Reply:
Yeah, it is spelling issue. But actually it is still dummy implementation. Let me fix it. Thanks.